examples/gallery/gallery.slint

```diff
@@ -1,160 +1,66 @@
-// Copyright © SixtyFPS GmbH <info@slint-ui.com>
```

Comment:
I suppose that changes to this file were not intended for this merge request? :)